ConsoleMenu/ItemsCollection.cs

```diff
@@ -97,6 +99,18 @@ internal bool IsSelected(MenuItem menuItem)
     return this.currentItemIndex == menuItem.Index;
   }
 
+  private static int GetSelectedIndex(char ch)
+  {
+    int index = ch switch
+    {
+      >= 'a' and <= 'z' => ch - 87,
+      >= 'A' and <= 'Z' => ch - 56,
```

Comment:
Let's avoid magical constants
```suggestion
      >= 'a' and <= 'z' => ch - 'a' + 10,
      >= 'A' and <= 'Z' => ch - 'A' + 10,
```

Reply:
Implemented